actions/ci-sonarqube-ts/action.yml

```diff
@@ -80,32 +80,32 @@ runs:
         workflow_conclusion: ""
         if_no_artifact_found: warn
 
-    - name: SonarQube scan without lint
-      if: inputs.include-lint == 'false'
-      uses: sonarsource/sonarqube-scan-action@aecaf43ae57e412bd97d70ef9ce6076e672fe0a9 # v2.3.0
-      with:
-        args: >
-          -Dsonar.junit.reportPaths=${{
-            inputs.junit-xml-report-path
-          }} -Dsonar.typescript.lcov.reportPaths=${{
-            inputs.typescript-lcov-report-path
-          }}
-      env:
-        SONAR_TOKEN: ${{ inputs.sonar-token }}
-        SONAR_HOST_URL: ${{ inputs.sonar-host-url }}
+    - name: Set SonarQube Report Paths
+      id: sonarqube_report_paths
+      shell: bash
+      run: |
+        sonarqube_coverage_report_paths=$(find -type f -name '${{ inputs.typescript-lcov-report-path }}' -printf "%p,")
```

Comment:
What does this line do?
What is expected fromat for passing multiple paths in the input? Is it a string with newlines as delimiter?

Reply:
it would be nice if you can document what is expected format of the input parameter in the input description

Reply:
It recursively looks for the file names specified in action input `typescript-lcov-report-path` (that corresponds to the report name) and assigns those values to this variable, used later by Sonar. It is tested [here](https://github.com/smartcontractkit/capabilities/actions/runs/11381438006/job/31662910820?pr=33#step:3:157), you may find paths delimited by coma (`%p,` where `p` is a path/value found)

Reply:
I have updated the input variable names, so users may now specify only report names instead of paths (the latter will be found automatically). If users specify paths, the action will fail, as it will look for file names.

Reply:
Thanks for explaining, so it assumes that all packages, modules in the project produce a report with the same file name? right?

It probably helps for some use cases, but for the other use cases the user of the workflow is not able to specify explicitly which exact report paths should be used. Can this be a problem for mono repos with variety of different frameworks and tools for generating such reports?